packages/dashboard/src/pages/SetupPersona/Welcome/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { DashboardRoutes, EnhanceableSite } from '@masknet/shared-base'
+import { makeStyles } from '@masknet/theme'
+import { Box, Checkbox, FormControlLabel, Typography } from '@mui/material'
+import { memo, useCallback, useState } from 'react'
+import { useNavigate, useSearchParams } from 'react-router-dom'
+import urlcat from 'urlcat'
+import { Services } from '../../../API.js'
+import { TermsAgreedContext } from '../../../hooks/useTermsAgreed.js'
+import { DashboardTrans, useDashboardI18N } from '../../../locales/index.js'
+import { SecondaryButton } from '../../../components/SecondaryButton/index.js'
+import { PrimaryButton } from '../../../components/PrimaryButton/index.js'
+
+import { SetupFrameController } from '../../../components/SetupFrame/index.js'
+import { Article } from './Article.js'
+
+const useStyles = makeStyles()((theme) => ({
+    title: {
+        fontSize: 36,
+        lineHeight: 1.2,
+        fontWeight: 700,
+    },
+    checkboxRow: {
+        marginTop: theme.spacing(1),
+        marginLeft: 0,
+    },
+    label: {
+        fontSize: 14,
+        color: theme.palette.maskColor.second,
+        marginRight: 0,
+        marginLeft: 0,
+    },
+    link: {
+        color: theme.palette.maskColor.main,
+        textDecoration: 'underline',
+    },
+    checkbox: {
+        margin: '-9px 12px -9px -9px',
+    },
+    buttonGroup: {
+        display: 'flex',
+        columnGap: 12,
+    },
+    policy: {
+        fontSize: 14,
+        lineHeight: '20px',
+        color: theme.palette.maskColor.second,
+        marginTop: 48,
+    },
+}))
+
+export const Welcome = memo(function Welcome() {
+    const [, setAgreed] = TermsAgreedContext.useContainer()
+    const [allowedToCollect, setAllowedToCollect] = useState(true)
+    const [params] = useSearchParams()
+    const navigate = useNavigate()
+
+    const handleAgree = useCallback(async () => {
+        if (allowedToCollect) {
+            Services.Settings.setTelemetryEnabled(true)
```

Comment:
```suggestion
            await Services.Settings.setTelemetryEnabled(true)
```

Reply:
Please note that this comment conflicts with https://github.com/DimensionDev/Maskbook/pull/9692#discussion_r1221661305.